src/stores/AppStore.js

```diff
@@ -687,6 +688,14 @@ export default types
       }
     }
 
+    function setUsers(users) {
+      self.users.replace(users);
+    }
+
+    function mergeUsers(users) {
+      self.setUsers(uniqBy([...getSnapshot(self.users), ...users], 'id'));
```

Comment:
what would happen if you just pass an array with non-unique users to mst? error or some merging policy? because maybe we can simplify it a bit.

Reply:
it will cause an error with `types.identifierNumber` duplication